QgisModelBaker/libqgsprojectgen/dbconnector/gpkg_connector.py

```diff
@@ -708,13 +717,23 @@ def create_basket(self, dataset_tid, topic):
                     topic
                 )
             try:
+                (
+                    next_id,
+                    fetch_and_increment_feedback,
+                ) = self._fetch_and_increment_key_object(self.tid)
```

Comment:
```suggestion
                next_id, fetch_and_increment_feedback = self._fetch_and_increment_key_object(self.tid)
```

No parenthesis required, I very much like those on single lines if that's acceptable for you

Reply:
For me yes, but not for black.

I'm semi-happy with black. Sometimes it gets not more readable with it.